include/swift/Driver/DependencyGraph.h

```diff
@@ -123,8 +123,9 @@ class DependencyGraphImpl {
   /// The set of marked nodes.
   llvm::SmallPtrSet<const void *, 16> Marked;
 
-  /// A list of all "external" dependencies that cannot be resolved just from
-  /// this dependency graph.
+  /// A list of all external dependencies that cannot be resolved
+  /// just from this dependency graph. They must be resolved from definitions
+  /// outside of this module.
```

Comment:
I'm not sure what this sentence means. The external dependencies are files, not names of declarations.

Reply:
It is trying to explain "external", as in "external to what?" I have tried some rewording. See what you think.